service/rtc/sfu.go

```diff
@@ -780,6 +787,58 @@ func (s *Server) CloseSession(sessionID string) error {
 	return nil
 }
 
+// collectSessionTracks gathers all available tracks from a session that should be delivered to the target session.
+func collectSessionTracks(senderSession *session, targetSession *session) ([]*webrtc.TrackLocalStaticRTP, []*webrtc.TrackRemote) {
```

Comment:
This is to address https://github.com/mattermost/rtcd/pull/173#discussion_r2025176872, especially since now it gets even more complicated. Still a somewhat involved method but a little better I suppose.